src/commands/autocomplete/values.js

```
@@ -0,0 +1,50 @@
+// @flow
+
+import {flags} from 'cli-engine-command'
+import HerokuCommand from 'cli-engine-heorku'
+import Plugins from '../../plugins'
+import ACCache from '../../cache'
+import path from 'path'
+
+export default class AutocompleteValues extends HerokuCommand {
+  static topic = 'autocomplete'
+  static command = 'values'
+  static description = 'generates autocomplete values'
+  static hidden = true
+  static flags = {
+    // don't require cmd or flag
+    // we want it to fail silently
+    // or autocomplete gets weird
+    cmd: flags.string({description: '', char: 'c'}),
+    flag: flags.string({description: '', char: 'f'})
+  }
+
+  async run () {
+    if (this.config.windows) {
+      this.out.error('Autocomplete is not currently supported in Windows')
+      return
+    }
+
+    if (this.flags.cmd) {
+      const plugins = new Plugins(this.out)
+      await plugins.load()
+      let Command = await plugins.findCommand(this.flags.cmd)
+      if (!Command || !this.flags.flag) return
```

you should throw an error if the command isn't found